Download table, updated phrasing, moved content for clarity

solaarTW opened the pull request:
Converted ASCII table to HTML table for future content and formatting improvements, reworded download table headers to match cell content and match cell descriptions in below paragraphs, removed duplicative text concerning downloadable content, moved statement on the download recommendation to the download table, for people with narrow screens I added user guide quick links to home pages of the wiki in case their monitor doesn't resolve the left-navigation bar nicely.

solaarTW committed:
Download table, unnecessary text, moved content for clarity

Converted ASCII table to HTML table for future content and formatting improvements, reworded download table headers to match cell content and match cell descriptions in below paragraphs, removed duplicative text concerning downloadable content, moved statement on the download recommendation to the download table, for people with narrow screens I added user guide quick links to home pages of the wiki in case their monitor doesn't resolve the left-navigation bar nicely.

ja-to commented on getting-started/download.md:
```diff
@@ -1,43 +1,70 @@
-| [WolvenKit](https://github.com/WolvenKit/WolvenKit/)                          | [![GitHub release (latest by date)](https://img.shields.io/github/v/release/WolvenKit/WolvenKit)](https://github.com/WolvenKit/WolvenKit/releases/latest)                                   | ![GitHub branch checks state](https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/check-only)    |
-
-The latest GitHub WolvenKit release will be marked on the left column with green text. Expand the **Assets** tab to view available downloads for any given WolvenKit release.
+The recommended download is the WolvenKit Nightly installer
```
The WolvenKit nightly build does not post an installer version. Additionally, it doesn't appear WolvenKit has had any installer release since 8.4.3.

Also while the current release doesn't support game version 1.6, generally speaking stable releases are preferred.

solaarTW replied:
``` Nightly build does not post an installer```
Unless someone is familiar with GitHub, the download table is very confusing. I tried making it a proper table that had headers for every column, but it looked really ugly. The current look is pretty, but the Package links look like they go to the zipped program, and the link-images look like they're just pictures of what the Package versions are. I agree that the ZIP is not technically an installer, but it ultimately removes confusion on the difference between the Source Package and the ZIP file (zip of what? the source, the program, other resources?).

``` generally speaking stable releases are preferred```
I totally agree! What can be done to get the stable version updated more often? Right now it straight up doesn't work and also the stable version's download page says, "❗INCOMPATABLE WITH GAME VERSION 1.6+ ❗Use WolvenKit development build 8.7 or higher for use with Cyberpunk game version 1.6".

ja-to commented on getting-started/download.md:
```diff
@@ -1,43 +1,70 @@
-### WolvenKit
+## System Requirements
 
-The main WolvenKit application in a portable zip file (Recommended)
```
The portable WolvenKit is the recommended (and only) way to install the application. Should not be removed.

solaarTW replied:
I think we're both referring to the ZIP file that contains the compiled WolvenKit program? I'm unsure how many people know what a "portable" is, this is the first time I've heard it in this context.

ja-to commented on getting-started/download.md:
```diff
@@ -1,43 +1,70 @@
 ### WolvenKit Console
 
-The standalone command-line based community modkit for Cyberpunk (Originally CP77 Tools)
+The standalone command-line based community modkit that replaced CP77 Tools.<br />
```
WolvenKIt CLI did not "replace" CP77 Tools; it is CP77 Tools.

solaarTW replied:
Cool, I can appreciate the author of CP77 Tools wanting their hard work carried on into WolvenKit.

rfuzzo replied:
I am the author of CP77Tools :) but yeah I’m fact CP77Tools is still used as term so jato is technically correct

ja-to commented on getting-started/download.md:
```diff
@@ -1,43 +1,70 @@
 ### Source Code
 
-A snapshot of the code used to compile this specific WolvenKit release. Only useful for developers.
+Only useful for developers, the source code is a snapshot used to compile the WolvenKit installer and WolvenKit console.
```
This is a comma splice. Also unnecessary/inaccurate as there is no installer.

solaarTW replied:
I'm not dying on a hill over a word, it can be called MickeyMouseLovesPeanuts for all I care :D I just want to make the instructions clearer and reduce confusion for new modders. The initial setup of WolvenKit adds %USERPFILE% roaming files and while the Plugin installs aren't technically a part of WolvenKit application, that process does add programs outside of wherever WolvenKit is extracted to. Unless someone is in the know, such as a developer, it looks and smells like its installing things.

solaarTW committed:
Updated: Download table, updated phrasing, moved content for clarity

Applied commented changes back into my edit from the original base.

Converted ASCII table to HTML table for future content and formatting improvements, reworded download table headers to match cell content and match cell descriptions in below paragraphs, removed duplicative text concerning downloadable content, moved statement on the download recommendation to the download table, for people with narrow screens I added user guide quick links to home pages of the wiki in case their monitor doesn't resolve the left-navigation bar nicely.

rfuzzo committed:
Merge branch 'main' into patch-1
